internal/mempool/mempool.go

```diff
@@ -2125,13 +2156,17 @@ func (mp *TxPool) VerboseTxDescs() []*VerboseTxDesc {
 // concurrent access as required by the interface contract.
 func (mp *TxPool) MiningDescs() []*mining.TxDesc {
 	mp.mtx.RLock()
+	defer mp.mtx.RUnlock()
+	return mp.miningDescs()
+}
+
+func (mp *TxPool) miningDescs() []*mining.TxDesc {
```

Comment:
Missing comment.

Reply:
Please define before use for consistency with the rest of the code base.

Reply:
Defined before use + added comment.